applications/protocols_serialization/server/prj.conf

```diff
@@ -13,6 +13,7 @@ CONFIG_NRF_RPC_ZCBOR_BACKUPS=1
 CONFIG_NRF_RPC_THREAD_STACK_SIZE=4096
 CONFIG_NRF_RPC_GROUP_INIT_WAIT_TIME=-1
 CONFIG_NRF_RPC_UART_TRANSPORT=y
+CONFIG_DK_LIBRARY=y
```

Comment:
Why is it required?

Reply:
I guess my comment (https://github.com/nrfconnect/sdk-nrf/pull/16151#discussion_r1654891873) is fair enough for this too.

Reply:
Mark as solved to not duplicate discussion referenced by link.